.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,44 @@
+name: Prepare Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Use this to provide a version, instead of deriving it from the changelog.
+        required: false
+        type: string
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      # The current realm studio release process keeps changes for the upcoming release
```

Comment:
I think this should go into the PR description instead of a comment this workflow.

Reply:
In general, it would be great with a bit of motivation and tradeoffs considered to be added to the description of this PR, if not for us then for the future maintainers.

Reply:
correct, initially I was going for offering both old and new changelog style in the PR, with comments explaining the difference - but on the way figured it's not worth the effort do add the legacy stuff, and just kept that bit - at which point the comment no longer made sense.